public/app/plugins/panel/alertlist/unified-alerting/UngroupedView.tsx

```diff
@@ -27,86 +31,176 @@ type UngroupedModeProps = {
   options: UnifiedAlertListOptions;
 };
 
+export const useGetExtraInfoForAlertItem = (returnTo: string, slicedRules: CombinedRuleWithLocation[]) => {
```

Comment:
Maybe rename this to something like `useGetRulesWithExtraInfo`? As it looks like it returns a set of rules, not just its info.

Reply:
💯